specs/double_then_triple.yaml

```diff
@@ -0,0 +1,58 @@
+name: timer
+doc: |-
+  start a timer with timer ID from input, when time is up, output a message
+patternsyntax: json
+nodes:
+  start:
+    branching:
+      type: message
+      branches:
+      - pattern: |
+          {"double":"?double"}
+        target: double
+        
+      - pattern: |
+          {"triple":"?triple"}
+        target: triple        
+  double:
+    action:
+      interpreter: ecmascript
+      source: |-
+        var d = _.bindings["?double"];
+        
+        delete _.bindings["?double"];
+       
+      
+        var f = parseInt(d);
+        if (isNaN(f)) {
+           _.out({"invalid duration":d});
+        } else {
+        _.out({"doubled":f*2});
+        _.bindings["?triple"]=f*2;
```

Comment:
Why adding `?tripble` binding here?  It will cause the pattern match for `{"triple":"?triple"}` only works for whatever value it stores.